features/networking/pod.feature

```diff
@@ -297,3 +297,46 @@ Feature: Pod related networking scenarios
       | type          | merge                                                         						   |
     Then the step should fail
     And the output should contain "endpoints "<%= cb.ping_pod.name %>" is forbidden: endpoint port TCP:22623 is not allowed"
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-21846
+  @admin
+  @destructive
+  Scenario: ovn pod can be scheduled even if the node taint to unschedule
+    Given the env is using "OVNKubernetes" networkType
+    And I store all worker nodes to the :nodes clipboard
+    #Tainting all worker nodes to NoSchedule
+    When I run the :oadm_taint_nodes admin command with:
+      | l         | node-role.kubernetes.io/worker |
+      | key_val   | key2=value2:NoSchedule         |
```

Comment:
Perhaps replace key2 with key21846 or something like that for easier debug ?

Reply:
Yep this will be useful. Done. Thanks